AutomatedTesting/Gem/PythonTests/Prefab/TestSuite_Periodic.py

```diff
@@ -7,73 +7,71 @@
 
 import pytest
 
-from ly_test_tools.o3de.editor_test import EditorSingleTest, EditorSharedTest, EditorParallelTest, EditorTestSuite
+from ly_test_tools.o3de.editor_test import EditorSingleTest, EditorBatchedTest, EditorParallelTest, EditorTestSuite
 
 
 @pytest.mark.SUITE_periodic
 @pytest.mark.parametrize("launcher_platform", ['windows_editor'])
 @pytest.mark.parametrize("project", ["AutomatedTesting"])
 class TestAutomationNoAutoTestMode(EditorTestSuite):
 
-    # Enable only -BatchMode for these tests. Some tests cannot run in -autotest_mode due to UI interactions
-    global_extra_cmdline_args = ["-BatchMode"]
+    global_extra_cmdline_args = ["--regset=O3DE/Preferences/Prefabs/EnableOverridesUx=true"]
```

Comment:
Does it mean after this PR we now run all prefab automated tests with the flag on by default?

Reply:
I think it's the way to go, however have only tested with the periodic suite so far. I'll have another PR ready hopefully by EoD to move a bunch of tests over to EditorBatchedTest, and will likely make this change the Prefab TestSuite_Main as well.

Reply:
@jckand-amzn , eventually overrides will be the default editing mechanism but switching all the tests right now to use override mode by default seems like the wrong thing to do now. Every editor user uses the features with the flag turned off. So, by turning this option on, we will be opening ourselves to being disconnected from how the majority of users use and see the editor. 

I recommend that we enable this flag only for the override tests for now and make it a global setting when we are ready to release the default override mode.